.travis.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  - docker
+before_install:
+  - docker build -f Dockerfile.test --target ci -t devops-ci-tests .
+script:
+  - docker run -e TRELLO_API_KEY=$TRELLO_API_KEY -e TRELLO_API_TOKEN=$TRELLO_API_TOKEN -e TRELLO_BOARD_ID=$TRELLO_BOARD_ID devops-ci-tests
```

Comment:
Is there a cleaner way of doing this?

Reply:
Well, there's the slight improvement of `-e TRELLO_API_KEY` instead of `-e TRELLO_API_KEY=$TRELLO_API_KEY`

How come you've included TRELLO_BOARD_ID? The e2e tests should be creating and destroying a temp board not using the "real" one